package.json

```diff
@@ -104,7 +104,7 @@
     "react-helmet": "^6.1.0",
     "react-icons": "^4.3.1",
     "react-is": "18.2.0",
-    "react-router-dom": "5.3.0",
+    "react-router-dom": "6.4.1",
```

Comment:
When updating, why don't you update to latest available version? v6.4.1 was released 8 months ago. Let's update to latest version available.

I will reuse this comment for another thing. After the commit where you update `react-router-dom`, there are three commits in which you do required migrations. You have divided work in areas where you make changes. That is good. There could be another approach, where you divide work in commits by relevant type of migration. For example, I see that you changed all `<Switch>` elements to `<Routes>`. The commit message could then point to the [exact section](https://reactrouter.com/en/main/upgrading/v5#upgrade-all-switch-elements-to-routes) in migration documentation.

But, you don't have to make this change, it's just how I would divide it. Maybe we should add some relevant excerpts from migration documentation to the commit message of 037ec60e917dbdd7550e3741513b8b06cdada3e6 and not have next three commits refer to this one as **previous commit**: "Please check the upgrading from v5 to v6 guide of the **previous commit** message to understand why we changing the structure"